pkg/cli/cmds/etcd_snapshot.go

```diff
@@ -2,38 +2,51 @@ package cmds
 
 import (
 	"github.com/rancher/k3s/pkg/cli/cmds"
+	"github.com/rancher/k3s/pkg/cli/etcdsnapshot"
 	"github.com/rancher/rke2/pkg/rke2"
 	"github.com/urfave/cli"
 )
 
+var k3sFlags = map[string]*K3SFlagOption{
```

Comment:
Why the breakout from the `var ()` block below - is this a gofmt thing?

Reply:
Oh I see, if you want to reference them in subsequent var definitions you have to define them earlier.

Reply:
Accounting for the coming PRs with the rest of the commands.